src/main/scala/com/linagora/openpaas/gatling/calendar/CalendarTemplateRequestsList.scala

```diff
@@ -166,49 +166,48 @@ object CalendarTemplateRequestsList {
+    //"/unifiedinbox.github/app/components/configuration/social-networks-item.controller.js",
+    //"/unifiedinbox.github/app/components/event/github-event.component.js",
+    //"/unifiedinbox.github/app/components/sidebar/sidebar-menu-item.component.js",
+    //"/unifiedinbox.github/app/components/sidebar/sidebar-menu.component.js",
+    //"/unifiedinbox.github/app/components/sidebar/sidebar-menu.controller.js",
+    //"/unifiedinbox.github/app/inbox.github.config.js",
+    //"/unifiedinbox.github/app/inbox.github.constants.js",
+    //"/unifiedinbox.github/app/inbox.github.run.js",
+    //"/unifiedinbox.github/app/providers/github-event-provider.service.js",
+    //"/unifiedinbox.github/app/services/github-api-events.service.js",
     "/calendar/components/fullcalendar/dist/locale-all.js",
-    "/unifiedinbox.github/app/services/github-api.service.js",
-    "/unifiedinbox.github/app/services/plugin/github-plugin.run.js",
-    "/unifiedinbox.twitter/app/inbox.twitter.app.js",
-    "/unifiedinbox.twitter/app/components/configuration/social-networks-item.component.js",
-    "/unifiedinbox.twitter/app/components/configuration/social-networks-item.controller.js",
-    "/unifiedinbox.twitter/app/components/list/inbox-list-item-content/inbox-list-item-content.component.js",
-    "/unifiedinbox.twitter/app/components/list/inbox-list-item-content/inbox-list-item-content.controller.js",
-    "/unifiedinbox.twitter/app/components/list/twitter-list-item.component.js",
-    "/unifiedinbox.twitter/app/components/sidebar/sidebar-menu-item.component.js",
-    "/unifiedinbox.twitter/app/components/sidebar/sidebar-menu.component.js",
-    "/unifiedinbox.twitter/app/components/sidebar/sidebar-menu.controller.js",
-    "/unifiedinbox.twitter/app/inbox.twitter.config.js",
-    "/unifiedinbox.twitter/app/inbox.twitter.constants.js",
+    //"/unifiedinbox.github/app/services/github-api.service.js",
+    //"/unifiedinbox.github/app/services/plugin/github-plugin.run.js",
+    //"/unifiedinbox.twitter/app/inbox.twitter.app.js",
+    //"/unifiedinbox.twitter/app/components/configuration/social-networks-item.component.js",
+    //"/unifiedinbox.twitter/app/components/configuration/social-networks-item.controller.js",
+    //"/unifiedinbox.twitter/app/components/list/inbox-list-item-content/inbox-list-item-content.component.js",
+    //"/unifiedinbox.twitter/app/components/list/inbox-list-item-content/inbox-list-item-content.controller.js",
+    //"/unifiedinbox.twitter/app/components/list/twitter-list-item.component.js",
+    //"/unifiedinbox.twitter/app/components/sidebar/sidebar-menu-item.component.js",
+    //"/unifiedinbox.twitter/app/components/sidebar/sidebar-menu.component.js",
+    //"/unifiedinbox.twitter/app/components/sidebar/sidebar-menu.controller.js",
+    //"/unifiedinbox.twitter/app/inbox.twitter.config.js",
+    //"/unifiedinbox.twitter/app/inbox.twitter.constants.js",
```

Comment:
Commented code deserves to be in the git history, not as a comment.

Can we remove it?